package.json

```diff
@@ -37,21 +37,19 @@
     }
   },
   "lint-staged": {
-    "**/*.{js,md}": [
+    "**/*.{js,ts,md}": [
       "prettier --write"
     ],
-    "*.js": "eslint --fix"
+    "*.{js,ts}": "eslint --fix"
+  },
+  "dependencies": {
```

Comment:
probably better optional?

Reply:
I thought about moving all optional to dev dependencies

Reply:
Probably except jest-dev-server, because this won't be installed by the user in their repo from my understanding.

Reply:
for me jest-dev-server is more a dependency instead of an optional or devdependency

Reply:
I suppose **jest-dev-server** shouldn't be installed if there is no need to use it

Reply:
but how should the user decide if he wants jest-dev-server or not?
Should he install it manually on his repository? Seems weird for me for such a small package. I would include it into dependency section if you don't mind.

Reply:
maybe you are right. It's not critical to install  jest-dev-server. So it's ok for me